sqlmesh/utils/date.py

```diff
@@ -96,6 +107,7 @@ def to_timestamp(value: TimeLike, relative_base: t.Optional[datetime] = None) ->
     return int(to_datetime(value, relative_base=relative_base).timestamp() * 1000)
 
 
+@ttl_cache(maxsize=1280000)
```

Comment:
ditto: magic number. Should this be default in the `ttl_cache`?

Reply:
there already is one, ttl_cache is time based, maxsize is pass through to the python cache method

Reply:
the current default is 60. Why have it if we override it everywhere. Should the default just be 1280000?